packages/nc-gui/components/account/UserList.vue

```diff
@@ -72,20 +78,30 @@ const updateRole = async (userId: string, roles: Roles) => {
   }
 }
 
-const deleteUser = async (userId: string) => {
+const deleteModalInfo = ref<UserType | null>(null)
+
+const deleteUser = async () => {
   try {
-    await api.orgUsers.delete(userId)
+    await api.orgUsers.delete(deleteModalInfo.value?.id as string)
     message.success(t('msg.success.userDeleted'))
+
     await loadUsers()
+
+    if (!users.value.length && currentPage.value !== 1) {
+      currentPage.value--
+      loadUsers(currentPage.value)
+    }
```

Comment:
On delete do we have to navigate back to previous page ??

Reply:
when in current page only one element is available and  when we delete that user we need to go back to previous page